README.md

```diff
@@ -58,6 +58,20 @@ ls -last /path/to/fedora_home
 
 For a detailed explanation of the configuration options have a look at [Application Configuration](https://wiki.lyrasis.org/display/FEDORA6x/Application+Configuration) in the Lyrasis Wiki.
 
+## Official Dockerhub Releases and Tags
+
+This repository also houses the GitHub actions and shell scripts which are responsible for updating the [Official fcrepo DockerHub repository](https://hub.docker.com/r/fcrepo/fcrepo/tags). There you will find tagged releases for your convenience following the common standard of:
+
+- `fcrepo/fcrepo:latest` : points to the most recent stable MAJOR.MINOR.PATCH release e.g. 'fcrepo/fcrepo:6.5.0'. Use this with care as it may jump MAJOR versions with large breaking changes major updates to core libraries and perhaps new ways of working.  
```

Comment:
We are **not** going to continue labelling things with `latest` because docker automatically labels things without a label as `latest` so it is not reliably consistent. 

So this line should be removed and replaced with a note clarifying that we will be phasing out the use of `latest` and people should instead rely on the below tags.